vimeo-networking/src/main/java/com/vimeo/networking/model/Membership.java

```diff
@@ -0,0 +1,95 @@
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * This object contains user membership information.
+ * <p>
+ * Created by brentwatson on 12/3/18.
+ */
+@SuppressWarnings("unused")
+public class Membership implements Serializable {
+
+    private static final long serialVersionUID = 2005778103318600521L;
+
+    /**
+     * The user's membership level
+     */
+    @SerializedName("display")
+    private String mDisplay;
```

Comment:
missing nullability annotation

Reply:
This field is not-nullable (per API docs). So I was following convention here. Seems like Nullable fields have `@Nullable` and other fields have no annotations.  If you add `@NotNull` then you need a constructor to enforce that fact otherwise you get IDE lint errors.